src/routes/api/user.routes.js

```diff
@@ -221,4 +222,72 @@ router.post('/signin', checkValidEmail, checkPassword, userController.signin);
  *  */
 
 router.delete('/logout', userController.logout);
+
+
+/**
+ * @swagger
+ *
+ * /api/v1/auth/forgot-password:
+ *   post:
+ *     security: []
+ *     summary: Reset password
+ *     description: users request to reset a password
+ *     tags:
+ *       - Users
+ *     requestBody:
+ *       content:
+ *         application/json:
+ *           schema:
+ *             type: object
+ *             properties:
+ *               email:
+ *                 type: string
+ *     produces:
+ *         application/json:
+ *           schema:
+ *             type: object
+ *             properties:
+ *               status:
+ *                 type: integer
+ *               message:
+ *                 type: string
+ *     responses:
+ *       200:
+ *         description: We have sent you an email
+ *  */
+router.post('/forgot-password', checkValidEmail, validateResult, userController.forgotPassword);
```

Comment:
```suggestion
router.put('/forgot-password', checkValidEmail, validateResult, userController.forgotPassword);
```

Reply:
Thank you @blaise82. I put `PUT` on the reset password route instead. Because it is where I am updating from